modules/openapi-generator/src/main/resources/kotlin-spring/optionalDataType.mustache

```diff
@@ -1 +1 @@
-{{^isFile}}{{{dataType}}}{{^required}}{{^defaultValue}}?{{/defaultValue}}{{/required}}{{/isFile}}{{#isFile}}{{#isArray}}Array<{{/isArray}}org.springframework.web.multipart.MultipartFile{{#isArray}}>{{/isArray}}{{^isArray}}{{^required}}?{{/required}}{{/isArray}}{{/isFile}}
\ No newline at end of file
+{{^isFile}}{{{dataType}}}{{^required}}{{^defaultValue}}?{{/defaultValue}}{{/required}}{{/isFile}}{{#isFile}}{{#isArray}}Array<{{/isArray}}org.springframework.web.multipart.MultipartFile{{#isArray}}>{{/isArray}}{{#isNullable}}?{{/isNullable}}{{/isFile}}
```

Comment:
what about optional schema in which isNullable is set to false? should it also generate the `?` at the end to make it nullable in kotlin?

Reply:
What do you mean exactly? 

Something equivalent to Optional<T> in Java?

Reply:
If both are managed by placeholders `required` and `isNullable` that's ok so ;)
If something (class, property,...) has `nullable` set to true, the type should have a `?`

Reply:
i'll file another pr to show i mean by optional after merging yours